Sprint-3/2-practice-tdd/count.test.js

```diff
@@ -17,8 +17,91 @@ test("should count multiple occurrences of a character", () => {
   expect(count).toEqual(5);
 });
 
+test("should count multiple occurrences of a character", () => {
+  const str = "bbb";
+  const char = "b";
+  const count = countChar(str, char);
+  expect(count).toEqual(3);
+});
+
+test("should count mutiple of character 'coco can have coconut'", () => {
+  const str = "coco can have coconut";
+  const char = "c";
+  const count = countChar(str, char);
+  expect(count).toEqual(5);
+});
+
+test("should count multiple occurrences of a character 'a' in 'I have an apple'", () => {
+  const str = "I have an apple";
+  const char = "a";
+  const count = countChar(str, char);
+  expect(count).toEqual(3);
+});
+
 // Scenario: No Occurrences
 // Given the input string str,
 // And a character char that does not exist within the case-sensitive str,
 // When the function is called with these inputs,
 // Then it should return 0, indicating that no occurrences of the char were found in the case-sensitive str.
+
+test("should return 0 when the character does not exist in the string", () => {
+  const str = "hello";
+  const char = "z";
+  const count = countChar(str, char);
+  expect(count).toEqual(0);
+});
+
+test("should return 0 when the 'c' does not exist in string 'I dont have an apple'", () => {
+  const str = "I dont have an apple";
+  const char = "c";
+  const count = countChar(str, char);
+  expect(count).toEqual(0);
+});
+
+// Scenario: Find character is longer than the input string
+// Given a string shorter than the findCharacter (e.g., 'a' vs 'abc'),
+// When the function is called,
+// Then it should return 0 because a multi-character string cannot match a single character position.
+
+test("should return 0 when findCharacter is longer than the input string", () => {
```

Comment:
I think it's a bit incorrect. This is related to my earlier comment for the `countChar` function. If str length is shorter than char length, the function should return an error. Returning 0 should be reserved only for the case when character is not found. Otherwise, you function logic becomes convoluted, you wouldn't know if it returned 0 because character is not found or because arguments were invalid. Do you see the problem with it?